### ISSUE TITLE
What about Twitter, I think it’s time to let in new users

### ISSUE THREAD

**Issue by VegasBtc** [thisrepo/dog#40]

**Comment by Cryptobertl** [comment]
Awesome idea I would say I agree

**Comment by Eriwa00** [comment]
Yeah Twitter is okay , I don’t really think telegram is necessary again because an added uniqueness of the meme is the the fact that we use git to sub for telegram , getting a telegram boost to get more people eyes on it won’t be a bad idea imo

**Comment by thisrepo** [comment]
made an official update on that topic: https://github.com/thisrepo/dog/issues/42
